platform/ethereum/domain.go

```diff
@@ -1,14 +1,33 @@
 package ethereum
 
 import (
-	CoinType "github.com/trustwallet/blockatlas/coin"
+	"strings"
+
+	"github.com/trustwallet/blockatlas/coin"
+	"github.com/trustwallet/blockatlas/pkg/address"
 	"github.com/trustwallet/blockatlas/pkg/blockatlas"
 	"github.com/trustwallet/blockatlas/pkg/errors"
 	"github.com/trustwallet/blockatlas/pkg/logger"
 	"github.com/trustwallet/blockatlas/platform/ethereum/ens"
-	AddressEncoder "github.com/trustwallet/ens-coincodec"
+	coincodec "github.com/trustwallet/ens-coincodec"
```

Comment:
are you serious ?
PLease, remove alias

Reply:
the module name has a `-`, is it ok to call?

Reply:
there is no `- `
it will be just coincodec package and that is it

Reply:
Unfortunately packagename with - ens-codec cannot be used without alias conveniently.  Without alias that's what happens:
spaces added
<img width="504" alt="image" src="https://user-images.githubusercontent.com/13562139/83878418-6e61c900-a73c-11ea-8b46-259e5537b69c.png">

alias added:
<img width="453" alt="image" src="https://user-images.githubusercontent.com/13562139/83878442-791c5e00-a73c-11ea-98c1-f44b394ccd99.png">

fails to compile:
```
# github.com/trustwallet/blockatlas/platform/ethereum
platform/ethereum/domain.go:65:22: use of package ens without selector
platform/ethereum/domain.go:65:42: multiple-value coincodec.ToString() in single-value context
```

Reply:
....

Reply:
https://github.com/trustwallet/blockatlas/pull/1132/commits/fc07b8d95e0cb0fda2de5d56802b4a25998ae862

Look at that please

Reply:
Let me try this 3rd time: goimports adds alias coincodec in this case automatically.  Please use goimports.

Reply:
OK, let's have it like this.  It will be changed next time when someone saves it with goimports though.